src/StatusQ/Popups/Dialog/StatusDialogFooter.qml

```diff
@@ -0,0 +1,52 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import ".."
+
+Rectangle {
+    id: root
+
+    property ObjectModel leftButtons
+    property ObjectModel rightButtons
+
+    color: Theme.palette.statusModal.backgroundColor
+    radius: 8
+    clip: true
+
+    height: implicitHeight
+    implicitHeight: layout.implicitHeight + layout.anchors.topMargin + layout.anchors.bottomMargin
+    implicitWidth: layout.implicitWidth + layout.anchors.leftMargin + layout.anchors.rightMargin
+
+    RowLayout {
+        id: layout
+
+        anchors {
+            fill: parent
+            margins: 16
+        }
+
+        Repeater {
+            model: root.leftButtons
+            onItemAdded: item.Layout.fillHeight = true
+        }
+
+        Item {
+            Layout.fillWidth: true
+        }
+
+        Repeater {
+            model: root.rightButtons
+            onItemAdded: item.Layout.fillHeight = true
+        }
+    }
+
+    StatusModalDivider {
+        y: 0.1 // HACK: when y == 0, then divider is not visible in some rare cases
```

Comment:
I guess it's due to the `height: implicitHeight` at the top and the clipping, worth trying out :)

Reply:
I will try, but don't see any logical reason how `height: implicitHeight` and `clip: true` can impact drawing the line at position `y: 0` :thinking:

Reply:
@caybro removing `clip: true` fixed the issue. I still don't get why, in this case the footer should never be clipped vertically.